services/ui-src/src/utils/validation/schemas.ts

```diff
@@ -116,10 +116,24 @@ export const date = () =>
 
 export const dateOptional = () =>
   string()
-    .typeError(error.INVALID_GENERIC)
+    .nullable()
     .test({
       message: error.INVALID_DATE,
-      test: (value) => dateFormatRegex.test(value!),
+      test: (value) => {
+        if (
+          value === null ||
+          value === undefined ||
+          isWhitespaceString(value)
+        ) {
+          return true;
+        } else if (typeof value === "string") {
```

Comment:
when is value not a string?

I feel this can be simplified down to
` return (
          value === null ||
          value === undefined ||
          isWhitespaceString(value) ||
          dateFormatRegex.test(value)
        );`

Reply:
Updated!